mmv1/third_party/terraform/services/networkconnectivity/resource_network_connectivity_group_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
```

Comment:
Do you mind removing the copyright? It will be added automatically.

Reply:
npnp, removed :)